gdp-src-build/conf/templates/minnowboard.local.conf

```diff
@@ -3,3 +3,5 @@ include templates/local.inc
 MACHINE = "intel-corei7-64"
 SDKIMAGE_FEATURES_append = " staticdev-pkgs"
 IMAGE_INSTALL_append = " mesa-megadriver"
+
+CUSTOM_IMAGE_FSTYPE ?= "ext4 wic.gz"
```

Comment:
Why both?

Reply:
As you have noticed minnowboard produces a `hddimg` which has a silent dependency on `ext4`. Unclear where the `hddimg` is added.

But `hddimg` is actually the correct image for minnowboard, as it is a EFI enabled image which is required for the hardware to boot so I will update to:

    CUSTOM_IMAGE_FSTYPE ?= "hddimg"